Resources/Prototypes/_NF/Entities/Objects/Weapons/Bombs/mines.yml

```
@@ -0,0 +1,737 @@
+# Base
+- type: entity
+  parent: [ BaseLandMine, BaseItem ]
+  id: NFConcealedMineTrigger
+  abstract: true
+  components:
+  - type: CollisionWake
+    enabled: false
+  - type: Physics
+    bodyType: Dynamic
+  - type: AmbientSound
+    volume: -15
+    range: 2
+    enabled: true
+    sound:
+      path: /Audio/Items/newton_cradle.ogg
+  - type: ExplodeOnTrigger
+  - type: DeleteOnTrigger
+  - type: OnUseTimerTrigger
+    useVerbInstead: true
+    beepInterval: .25
+    beepSound:
+      path: /Audio/Effects/beep_landmine.ogg
+      params:
+        volume: 1
+    examinable: false
+  - type: TriggerOnProximity
```

```suggestion
  - type: TriggerOnProximity
    triggerSpeed: 0
```
With this it'll be an actual proximity bomb, and not need you to walk quickly over it.